include/yaml_cpp_catkin/yaml_eigen.h

```diff
@@ -122,12 +122,19 @@ struct convert<Eigen::Matrix<Scalar,  Rows,  Cols,  Align,  RowsAtCompileTime,
   static bool decode(const Node& node, Eigen_Type_& rhs){
     if(!node.IsSequence())
       return false;
-    else if(!node[0].IsSequence())
-      return decode_1d(node, rhs);
+    if(node.size() > 0)
+    {
+      if(!node[0].IsSequence())
+        return decode_1d(node, rhs);
```

Comment:
Please add braces also in case of one-line if/else.

Reply:
there seem to be other places in the code with same issue